org-eclipse-che-jdt-ls-extension/pom.xml

```diff
@@ -11,205 +11,211 @@
-                        </execution>
-                        <execution>
-                            <id>dependency</id>
-                            <phase>validate</phase>
-                            <goals>
-                                <goal>enforce</goal>
-                            </goals>
-                            <configuration>
-                                <skip>true</skip>
-                            </configuration>
-                        </execution>
-                    </executions>
-                </plugin>
-                <plugin>
-                    <groupId>org.eclipse.tycho</groupId>
-                    <artifactId>tycho-packaging-plugin</artifactId>
-                    <version>${tycho.version}</version>
-                    <dependencies>
-                        <dependency>
-                            <groupId>org.eclipse.tycho.extras</groupId>
-                            <artifactId>tycho-sourceref-jgit</artifactId>
-                            <version>${tycho.extras.version}</version>
-                        </dependency>
-                    </dependencies>
-                    <configuration>
-                        <format>yyyyMMdd-HHmm</format>
-                        <sourceReferences>
-                            <generate>${tycho.generateSourceReferences}</generate>
-                        </sourceReferences>
-                    </configuration>
-                </plugin>
-                <plugin>
-                    <groupId>org.eclipse.tycho</groupId>
-                    <artifactId>tycho-surefire-plugin</artifactId>
-                    <version>${tycho.version}</version>
-                    <configuration>
-                        <useUIHarness>true</useUIHarness>
-                        <argLine>${tycho.test.jvmArgs}</argLine>
-                        <!-- kill test JVM if tests take more than 1 minute (60 seconds) to finish -->
-                        <forkedProcessTimeoutInSeconds>60</forkedProcessTimeoutInSeconds>
-                    </configuration>
-                </plugin>
-            </plugins>
-        </pluginManagement>
-        <plugins>
-            <plugin>
-                <groupId>org.eclipse.tycho</groupId>
-                <artifactId>tycho-maven-plugin</artifactId>
-                <version>${tycho.version}</version>
-                <extensions>true</extensions>
-            </plugin>
-            <plugin>
-                <groupId>org.eclipse.tycho</groupId>
-                <artifactId>tycho-compiler-plugin</artifactId>
-                <version>${tycho.version}</version>
-                <configuration>
-                    <useProjectSettings>false</useProjectSettings>
-                    <compilerArgs>
-                        <arg>-err:-forbidden</arg>
-                        <arg>-warn:-forbidden</arg>
-                    </compilerArgs>
+<!-- Copyright (c) 2012-2017 Red Hat, Inc. All rights reserved. This program 
```

Comment:
it seems format of the file has been changed (space <--> tabs) so it's hard to check what is different